app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+
+
+    var watch = function(context, callback, opts ) {
+      return DataService.watch(mobileclientVersion, context, callback, opts);
+    };
+
+    var getMobileClients = function(namespace) {
+      return DataService.list(mobileclientVersion, {namespace: namespace})
+        .then(function(clients) {
+          return clients.by("metadata.name");
+        });
+    };
+
+
+    var filterExcluded = function(serviceName, mobileClients) {
+      return _.filter(mobileClients, function(client) {
+        return _.includes(client.spec.excludedServices, serviceName);
+      });
+    };
+
+    var filterNotExcluded = function(mobileClients, apiObject) {
+      var serviceId = _.get(apiObject, 'metadata.name', '');
```

Comment:
Can we make signature of `filterExcluded` and `filterNotExcluded` to be the same, 
either both would take `apiObject` or `serviceName/serviceId` and same order of args?

Reply:
:+1: